src/VisualStudio/IntegrationTest/TestUtilities/InProcess/ErrorList_InProc.cs

```diff
@@ -30,15 +30,20 @@ public void WaitForNoErrorsInErrorList()
             }
         }
 
-        public void NavigateToErrorListItem(int itemIndex)
+        public ErrorListItem NavigateToErrorListItem(int itemIndex, __VSERRORCATEGORY minimumSeverity = __VSERRORCATEGORY.EC_WARNING)
         {
-            var errorItems = GetErrorItems().AsEnumerable().ToArray();
+            var errorItems = GetErrorItems()
+                .AsEnumerable()
+                .Where(e => ((IVsErrorItem)e).GetCategory() <= minimumSeverity)
+                .ToArray();
             if (itemIndex > errorItems.Count())
             {
                 throw new ArgumentException($"Cannot Navigate to Item '{itemIndex}', Total Items found '{errorItems.Count()}'.");
```

Comment:
Consider revising this to "total items with severity <blah> found" so it's clear that filtering happened.

Reply:
I'm planning to keep this for two reasons:

1. The filtering always used to happen, and the only change is the filtering is now explicit in code
2. This issue is acutely impacting integration test reliability and I'd like to get it merged earlier for stabilization